aws/ec2/resources/amazon-linux2.sh

```diff
@@ -7,6 +7,9 @@ export HOME=/root
 export DOPPLER_TOKEN="${doppler_service_token}"
 export GIT_SHA="${git_sha}"
 
+# Add Doppler token to root env vars. Mainly for remote testing purposes
+echo "export DOPPLER_TOKEN=${doppler_service_token}" >> $HOME/.bashrc
```

Comment:
Is there a way we can use this token without exposing it as an environment variable?  Additionally, I'd expect the user's `.bashrc` to be world readable, meaning any user on the machine can access it. If persistence of the token is required, we can save it directly with `doppler configure service token $TOKEN`. This will write it to the user's Doppler config, which is only readable by the user (perms `0600`).